go/apps/api/routes/v2_ratelimit_set_override/handler.go

```diff
@@ -146,6 +142,8 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
 			return "", err
 		}
 
+		h.RatelimitNamespaceByNameCache.Remove(ctx, namespace.Name)
```

Comment:
instead of removing, it would be nice if we just revalidate, but that's probably a bit annoying to do cause the cache itself doesn't know how to revalidate.

that's something I thought about before and I'm torn making the cache aware of how to revalidate data, would make stuff like this or automatic periodic revalidation possible, but also couple es the cache with specific data-loading logic :/

Reply:
@coderabbitai please create an issue for us to revisit this at some point.

Dont assign it to anyone and label it with coreTeam for now thanks.